src/sidebar/components/annotation-action-bar.js

```diff
@@ -10,9 +10,24 @@ import { withServices } from '../util/service-context';
 import AnnotationShareControl from './annotation-share-control';
 import Button from './button';
 
+/** @typedef {import("../../types/api").Annotation} Annotation */
+
+/** @typedef {import('../../types/config').HostConfig} HostConfig */
+
+/**
+ * @typedef AnnotationActionBarProps
+ * @prop {Annotation} annotation - The annotation in question
+ * @prop {(e: Event) => any} onReply - Callbacks for when action buttons are clicked/tapped
```

Comment:
The `onReply` callback is used only in one place and the `Event` argument is not used, so we could potentially remove that argument for the time being.